ably.d.ts

```diff
@@ -210,6 +210,19 @@ declare namespace Types {
      */
     useBinaryProtocol?: boolean;
 
+    /**
+     * Override the url used by the realtime client to check if the internet is available.
+     *
+     * Should return a success response when available.
+     */
+    internetAvailabilityUrl?: string;
+
+    /**
+     * Disable the check used by the realtime client to check if the internet
+     * is available before connecting to a fallback host.
+     */
+    disableConnectivityCheck?: boolean;
```

Comment:
Perhaps match the naming with the URL option, and call it `disableInternetAvailabilityCheck`?

Reply:
As long as we're consistent, I'm indifferent 🙂

Reply:
done